version: introduce Lua module (#10717)
tarantool/tarantool: changelogs/unreleased/gh-10521-introduce-version-module.md

> ## feature/core

Reviewer: Looks more like `feautre/lua` to me.

Author: Sure, done

## feature/core

> * Introduced `version` Lua module, which supports creating and
>   comparing schema version (aka `dd_version` in `box.status`, aka
>   `box.space._schema:get('version')`). Intended to ease the use
>   of blocked features before `box.schema.upgrade` (gh-10521).

Reviewer 2: ```suggestion
* Introduced the `version` Lua module, which enables creation and
  comparison of schema version objects (values of `dd_version` in
  `box.status` or `box.space._schema:get('version')`). The module
  eases the use of features blocked before `box.schema.upgrade`
  (gh-10521).
```
